hack/build/generate-manifests.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+#Licensed under the Apache License, Version 2.0 (the "License");
+#you may not use this file except in compliance with the License.
+#You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#Unless required by applicable law or agreed to in writing, software
+#distributed under the License is distributed on an "AS IS" BASIS,
+#WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#See the License for the specific language governing permissions and
+#limitations under the License.
+
+set -euo pipefail
+
+script_dir="$(readlink -f $(dirname $0))"
+source "${script_dir}"/common.sh
+source "${script_dir}"/config.sh
+
+set -x
+
+templates="$@"
+if [ -z "${templates}" ]; then
+    templates="$(find "${BUILD_DIR}/manifest-templates/" -name *.in -type f)"
+fi
+
+generator="${BIN_DIR}/manifest-generator"
+outDir="${OUT_DIR}/generated/manifests"
+
+mkdir -p "${outDir}"
+
+(cd "${CDI_DIR}/tools/manifest-generator/" && go build -o "${generator}" ./...)
+
+for tmpl in ${templates}; do
+    tmpl=$(readlink -f "${tmpl}")
+    base=$(basename "${tmpl}")
+    outFile=${base%.in}
```

Comment:
so output file, which is a manifest, is the same name as the template without the _,in_ suffix. Ok